index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <main>
+        
+            <nav>
+                <div class="logo">
+                    <p>My Logo</p>
+                </div>
+                
+                <div class="menu-icon">
+                    <img src="images/menu-icon.svg" alt="">
+                </div>
+                
+              
+            </nav>
+        <section class="first-section">
+            <header>
+                <h1>I'm Abigael <br> Glad to see <br> you</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </header>
+    
+            <div class="socials">
+                <p>let's connect</p>
+                <div class="social-platforms">
+                    <a href="#"><img class="social" src="images/social-media-icons.png" alt="" aria-hidden="true"></a>
+                </div>   
+            </div>
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please place some margin from the top for the main section content and make sure it matches the design with the template. Right now it's overlapping the background Images and not looking good.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165054628-2a648e9d-3ed7-463b-a5ea-3c2ad23bf86c.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165054035-2d455cad-0b1d-44cb-983b-952f19950630.png)

Reply:
Thank you. I have made changes